lib/src/main/kotlin/com/swmansion/starknet/account/Account.kt

```diff
@@ -157,7 +157,7 @@ interface Account {
         calldata: Calldata,
         salt: Felt,
         params: DeployAccountParamsV3,
-        forFeeEstimate: Boolean = false,
+        forFeeEstimate: Boolean,
```

Comment:
I think we should introduce overloaded method though

Reply:
Sure, I added `signDeployAccountV3`, `signDeclareV2` and `signDeclareV3` without `forFeeEstimate` param.